pkg/addons/assets.go

```diff
@@ -27,15 +27,15 @@ import (
 func bindataRead(data []byte, name string) ([]byte, error) {
 	gz, err := gzip.NewReader(bytes.NewBuffer(data))
 	if err != nil {
-		return nil, fmt.Errorf("read %q: %w", name, err)
+		return nil, fmt.Errorf("read %q: %v", name, err)
```

Comment:
I think you have to update `golanci-lint`

Reply:
Good call, thanks. I verified that there are no generated file changes after I updated deps.